### PR TITLE
gh-38: Make Makefile following FOSS

### PR DESCRIPTION
**Pull request by qexat**
Co-authored-by: MithicSpirit <rpc01234@gmail.com>

# Description

Makes Makefile targets phony

Fixes gh-38

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

n/a

**Test Configuration**:

n/a

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added the changes to the [changelog](/docs/CHANGELOG.md)
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by qexat** [comment]
@MithicSpirit please review

**Comment by MithicSpirit** [comment]
I've usually seen it where where `.PHONY` is at the end and has all of the targets (e.g. `.PHONY: all setup dotenv`) but it shouldn't matter.

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
+
+.PHONY: setup
 setup:
 	poetry shell
 	poetry update
 	poetry install
 
+.PHONY: dotenv
 dotenv:
 	cp templates/.env .env
```
[the remark this answers is not in the document]
Comment (MithicSpirit) [reply Makefile:12]: yes

**Comment by qexat** [comment]
should be fine by now

**Pull request merged**